client/my-sites/checkout/composite-checkout/components/prepurchase-notices/index.jsx

```diff
@@ -1,13 +1,13 @@
 import {
 	getProductFromSlug,
-	isJetpackAntiSpam,
 	isJetpackAntiSpamSlug,
-	isJetpackBackup,
 	isJetpackBackupSlug,
 	isJetpackPlanSlug,
-	isJetpackScan,
 	isJetpackScanSlug,
-	planHasFeature,
+	getAllFeaturesForPlan,
+	// eslint-disable-next-line import/named
```

Comment:
I think this line will become irrelevant once the PR is merged and the package built. You could make the package changes in a different PR, or delete that line later.

Reply:
yeah it's ok after I rebuilt earlier